core/modules/installer/installer.browser.inc

```diff
@@ -573,3 +573,36 @@ function installer_browser_check_zip_loaded() {
   }
   return $zip_loaded;
 }
+
+/**
+ *
+ * Checks if relevant system folder is writable
+ *
+ * @param $type
+ *   Type of project to view. Must be one of 'module', 'theme' or 'layout'.
+ */
```

Comment:
Thanks for including documentation!  Removing the blank line at 538 makes the docblock more consistent with the rest of Backdrop.